cmd/sponge/cmd/import.go

```diff
@@ -47,13 +49,13 @@ func init() {
 	RootCmd.PersistentFlags().StringVar(&queryFlag, "query", defaultQuery, "query on the external table (where condition on mysql, query document on mongodb). It only works with a specific --type. Example updated_date >'2003-12-31 01:02:03'")
 
 	RootCmd.PersistentFlags().BoolVar(&importonlyfailedFlag, "onlyfails", defaultImportonlyfailed, "import only the the records that failed in the last import(default is import all)")
-
-	RootCmd.PersistentFlags().StringVar(&localErrorsDBFlag, "errors", defaultlocalErrorsDB, "set the file path for the report on errors (default is report.db)")
+	RootCmd.PersistentFlags().BoolVar(&reportErrorsFlag, "report", defaultReportErrors, "create report on records that fail importing (default is do not report)")
+	RootCmd.PersistentFlags().StringVar(&localErrorsDBFlag, "filepath", defaultlocalErrorsDB, "set the file path for the report on errors (default is report.db)")
 
 	RootCmd.AddCommand(importCmd)
 }
 
 func addImport(cmd *cobra.Command, args []string) {
 
-	sponge.Import(limitFlag, offsetFlag, orderbyFlag, queryFlag, typeFlag, importonlyfailedFlag, localErrorsDBFlag)
+	sponge.Import(limitFlag, offsetFlag, orderbyFlag, queryFlag, typeFlag, importonlyfailedFlag, reportErrorsFlag, localErrorsDBFlag)
```

Comment:
Style suggestion for future refactoring:

Instead of passing each flag separately into Import (making the func profile very big), consider creating a config struct and setting all flags into it.  This way all config options can be easily passed wherever they're needed.

Another method is to create a config package that owns all the config.  Doing so allows you to import sponge/config from anywhere and have it available globally.

Not sure which best fits here.